pydantic/fields.py

```diff
@@ -434,10 +434,10 @@ def merge_field_infos(*field_infos: FieldInfo, **overrides: Any) -> FieldInfo:
             field_info = copy(field_infos[0])
             field_info._attributes_set.update(overrides)
 
-            default_override = overrides.pop('default', PydanticUndefined)
+            default_override = overrides.pop('default', _default_sentinel)
```

Comment:
We can use `_Unset` from the same module instead

Reply:
Actually, that doesn't work...